src/iconsManifest/supportedExtensions.ts

```diff
@@ -2898,8 +2898,8 @@ export const extensions: IFileCollection = {
     {
       icon: 'nuxt',
       extensions: [],
-      filenamesGlob: ['nuxt.config'],
-      extensionsGlob: ['js', 'ts'],
+      filenamesGlob: ['nuxt.config', '.nuxtrc', '.nuxtignore'],
```

Comment:
Regarding both `.nuxtrc` and `.nuxtignore` files, it seems that they should work without extension: https://nuxtjs.org/docs/configuration-glossary/configuration-ignore/

Do  you have any other link where it states that they should be used with extension?

Reply:
yeah they should work without extension, not sure what I was thinking there.